app/views/trips/_trip.html.erb

```diff
@@ -6,4 +6,5 @@
   <td><%= l(trip.start_date) %></td>
   <td><%= l(trip.end_date) %></td>
   <td><%= trip.school_name %></td>
+  <td><%= link_to t("trips.index.destroy"), trip_path(trip), method: :delete, class: "list-button" %></td>
```

Comment:
Line length??

Reply:
Oh right, weird rubocop didn't complain about it... 🤔 
fixed in sha: `cdeb9b4`